static/js/stream_settings_ui.js

```diff
@@ -481,6 +497,7 @@ export function redraw_left_panel(left_panel_params = get_left_panel_params()) {
     }
     maybe_reset_right_panel();
 
+    render_on_empty_streams({isEmpty: buckets.other.length === stream_ids.length});
```

Comment:
We can directly check `hidden_ids.length` here, would be more easy to understand for someone who is reading the code. Also, we generally not use camel case variables in JS code.